tasks/watch.cr

```diff
@@ -38,10 +41,12 @@ module Sentry
       end
 
       self.successful_compilations += 1
-      if successful_compilations == 1
-        start_browsersync
-      else
-        reload_browsersync
+      if @reload_browser
+        if successful_compilations == 1
+          start_browsersync
+        else
+          reload_browsersync
+        end
```

Comment:
I think extracting a `reload_or_start_browser_sync` method would be nice so this is cleaned up a bit